### PR TITLE
[bug 1153120] show linkedin username or old url

### PR DESCRIPTION
**Pull request by Mte90**

### CODE REVIEW

File: mozillians/groups/tasks.py [hunk 1]
```diff
@@ -158,6 +158,7 @@ def notify_membership_renewal():
 
     for group in groups:
         curator_ids = group.curators.all().values_list('id', flat=True)
+        last_update = datetime.now() - timedelta(days=group.invalidation_days)
```
Comment (akatsoulas) [inline mozillians/groups/tasks.py]: What's the purpose of this line here?

Comment (Mte90) [inline mozillians/groups/tasks.py]: I think that I got that during the merge explained here https://github.com/mozilla/mozillians/pull/1593 but I think that is not necessary. I will remove it asap

Comment (Mte90) [inline mozillians/groups/tasks.py]: ok the new pr contain only the code of the pr :-)

File: mozillians/phonebook/validators.py [hunk 2]
```diff
@@ -24,6 +24,16 @@ def validate_twitter(username):
     return username
 
 
+def validate_linkedin(url):
+    '''Return the linkedin username from the url or the link'''
```
Comment (comzeradd) [inline mozillians/phonebook/validators.py]: Minor fix: for consistency reasons, could you change this docstring here to use same syntax format as the others?

**Comment by Mte90** [comment]
Unit test added, I am waiting a confirm for that before merge again the commits :-)

**Comment by comzeradd** [comment]
Thanks @Mte90, it looks good. Please go ahead and squash the commits.

**Comment by Mte90** [comment]
Done!

**Comment by akatsoulas** [comment]
Merging since this is approved.

**Pull request merged**